<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMainAlarm</class>
 <widget class="QWidget" name="FormMainAlarm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FormMainAlarm</string>
  </property>
  <property name="styleSheet">
   <string notr="true">FormMainAlarm {
background: rgb(224,160,128);
}</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>进料斗</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>称重斗</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>斗置零</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>20</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>故障清除</string>
   </property>
  </widget>
  <widget class="QWidget" name="dou_widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>1004</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QWidget {
background: white;
border: 1px solid blue;
border-radius: 20px;
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="dou_bg">
      <property name="minimumSize">
       <size>
        <width>720</width>
        <height>340</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>720</width>
        <height>340</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background: gray;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_hide">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>10</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
